Prox/Prox/Models/Place.swift

```diff
@@ -451,25 +472,30 @@ struct OpenHours {
     }
 
     func nextOpeningTime(forTime time: Date) -> String? {
-        guard let openingTime = getOpeningTime(forDate: time) else {
-                return nil
-        }
+        guard let allOpeningTimes = getOpeningTimes(forDate: time),
+            let openingTime = getNextOpeningPeriod(fromOpeningPeriods: allOpeningTimes, forDate: time) else {
```

Comment:
Could we simplify this by  appending tomorrow's earliest opening time to `allOpeningTimes` returning `getNetxOpeningPeriod` to that?